onnxruntime/core/platform/env.cc

```diff
@@ -19,6 +19,20 @@ limitations under the License.
 
 namespace onnxruntime {
 
+std::ostream& operator<<(std::ostream& os, const ThreadOptions::ThreadAffinity& aff) {
+  os << "{";
+  std::copy(aff.logical_proc_ids.cbegin(), aff.logical_proc_ids.cend(), std::ostream_iterator<int>(os, " "));
```

Comment:
>std::copy(aff.logical_proc_ids.cbegin(), aff.logical_proc_ids.cend(), std::ostream_iterator<int>(os, " ")); [](http://example.com/codeflow?start=2&length=107)

nit: this leaves an extra " " before then ending "}"
another approach may be that we print to cend-1 in std::copy, then print back() in the following line.

Reply:
Further, coma as a split by more intuitive than space.

Reply:
This is Okay for error only case.